doc/Architecture/arc-sec_x_road_security_architecture.md

```diff
@@ -117,7 +121,7 @@ For compliance with the security principle of integrity, the objective is to ens
 
 X-Road incorporates a public key infrastructure (PKI) whereby a certification authority (CA) issues authentication certificates to Security Servers and signing certificates to X-Road member organisations. The CA processes certificate signing requests conforming to \[[PKCS10](#Ref_PKCS10)\].
 
-All X-Road messages are signed by the signing key of the organisations that send the messages and all messages are logged. Message logging is enabled by default. This means that both message headers and message bodies are logged. Logging of message bodies may be disabled on Security Server level or for selected subsystems. The logs are stored as plaintext on the Security Server.
+All X-Road messages are signed by the signing key of the organisations that send the messages and all messages are logged. Message logging is enabled by default. This means that both message headers and message bodies are logged. Logging of message bodies may be disabled on Security Server level or for selected subsystems. The logs are stored as plaintext on the Security Server. It's also possible to install a Security Server with the message log add-on fully disabled.
```

Comment:
> The logs are stored as plaintext on the Security Server.

This should be updated, e.g.:

> By default, the logs are stored as plaintext on the Security Server, but encryption can be enabled by the Security Server administrator.

Reply:
Updated in https://github.com/nordic-institute/X-Road/pull/1038/commits/60cbdbc51130c8ae6b3aab8e0e143df02b9f5d35